runtime/src/fips.rs

```diff
@@ -1,18 +1,61 @@
 // Licensed under the Apache-2.0 license
+
 use caliptra_common::cprintln;
 use caliptra_drivers::CaliptraError;
 use caliptra_drivers::CaliptraResult;
 use caliptra_registers::mbox::enums::MboxStatusE;
+use zerocopy::{AsBytes, FromBytes};
 
 use crate::Drivers;
 
-/// Fips command handler.
 pub struct FipsModule;
 
+#[repr(C)]
+#[derive(Clone, Debug, Default, AsBytes, FromBytes)]
+pub struct VersionResponse {
+    pub mode: u32,
+    pub fips_rev: [u32; 3],
+    pub name: [u8; 12],
+}
+
+impl VersionResponse {
+    pub const NAME: &str = "Caliptra RTM";
```

Comment:
You should be able to do

```rust
pub const NAME: [u8; 12] = *b"Caliptra RTM";
```

And then you should be able to avoid the copy and can just return

```rust
Self{
    ...
    name: NAME,
    ...
}
```